src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  BackgroundBlurOptions,
+  BackgroundBlurVideoFrameProcessor,
+  BackgroundFilterSpec,
+  ConsoleLogger,
+  DefaultVideoTransformDevice,
+  Device,
+  LogLevel,
+  NoOpVideoFrameProcessor,
+  VideoFrameProcessor,
+} from 'amazon-chime-sdk-js';
+import React, {
+  createContext,
+  FC,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+
+interface Props {
+  /** The spec defines the assets that will be used for adding background blur to a frame */
```

Comment:
nit: End the sentence with a dot.

Should we link to JS SDK documentation on this once JS SDK is released?